lib/engine.js

```diff
@@ -78,34 +77,38 @@ Engine.prototype.start = function (callback) {
   });
   this.mailer = new Mailer(this._config.mailer);
   this.network = new ComplexClient(this._config.complex);
-  this.contracts = new storj.StorageManager(
-    new MongoDBStorageAdapter(this.storage),
-    { disableReaper: true }
-  );
-  this.redis = require('redis').createClient(this._config.redis);
-  this.redis.on('ready', () => {
-    log.info('connected to redis');
-    pow.checkInitTarget(this.redis, (err) => {
-      if (err) {
-        log.error('unable to initialize pow settings', err);
-      }
+  // console.log('require sergio', MongoDBStorageAdapter);
+  setTimeout(() => {
```

Comment:
Why this timeout is here?

Reply:
Accidentally pushed several changes here, will be fixed this week